test/fixtures/dubhash-uhash/test02.result

```diff
@@ -3,11 +3,11 @@
   "hashes_ffm": [
     {
       "format": 1,
-      "data": "k7iFrf4NoInN9jSQT9WfcQ"
+      "data": "aT6a+E09/MceZA4AW9xeLg"
     },
     {
       "format": 3,
-      "data": "93b885adfe0da089cdf63490"
```

Comment:
The `test*.result` files are not being invoked at all, I think we can delete - unless they have a purpose that I am not aware of. I've check it after seen the differences on the data values, that at least for `format 1` should remain the same as we are not doing any code changes in `dubhash` algo but just introducing `uhash` as an alternative